sunpy/map/mapbase.py

```diff
@@ -605,6 +721,16 @@ def rotation_matrix(self):
         """
         Matrix describing the rotation required to align solar North with
         the top of the image.
+         .. rubric:: Metadata keywords:
+
+        * ``PC1_1``
```

Comment:
These are an either or, and also normally written as `PCi_j` and `CDi_j` to cover all four. i.e it should be a first available with two entries.